TestResultSummaryService/parsers/BenchmarkParser.js

```diff
@@ -86,10 +87,19 @@ class BenchmarkParser extends Parser {
                 isValid = false;
             }
 
-            // Parse benchmark product
-            if ( ( curRegexResult = benchmarkProductRegex.exec( curItr.value ) ) !== null ) {
-                curBenchmarkProduct = curRegexResult[1];
+            // Parse Java version
+            if ( ( curRegexResult = javaVersionRegex.exec( curItr.value ) ) !== null ) {
+                curJavaVersion = curRegexResult[1];
             }
+            curTestData["javaVersion"] = curJavaVersion;
+
+            if ( ( curRegexResult = javaBuildDateRegex.exec( curJavaVersion ) ) !== null ) {
+                curJavaBuildDate = curRegexResult[1];
+            }
+            // Set benchmark product
+            curBenchmarkProduct = curJavaBuildDate;
+            
+            curTestData["jdkBuildDateUnixTime"] = this.convertBuildDateToUnixTime( curJavaBuildDate );
```

Comment:
If we change the value of `BenchmarkProduct` to `curJavaBuildDate`, do we need to save the `curJavaBuildDate` in `curTestData["jdkBuildDateUnixTime"]` again?

Also with this change, `benchmarkProductRegex` is no longer needed.

Reply:
Yeah we don't need it!  Removed `benchmarkProductRegex`. Thanks!